Corona-Warn-App/src/main/res/values/styles.xml

```diff
@@ -63,17 +62,26 @@
         <item name="navigationContentDescription">@string/accessibility_close</item>
     </style>
 
+    <style name="TextInputLayoutTheme" parent="Widget.MaterialComponents.TextInputLayout.FilledBox">
+        <item name="boxBackgroundColor">@color/colorContactDiaryListItem</item>
+        <item name="android:textColorHint">@color/colorTextPrimary2</item>
+        <item name="hintTextColor">@color/colorTextPrimary2</item>
+        <item name="boxCornerRadiusBottomEnd">@dimen/spacing_tiny</item>
```

Comment:
I think the corner radius is a bit too big of the textfields? `spacing_mega_tiny` looks more like the designs IMO

Reply:
I think you're right. Haven't noticed this.